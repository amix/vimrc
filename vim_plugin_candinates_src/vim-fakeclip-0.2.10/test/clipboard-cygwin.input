" Test: vim-fakeclip clipboard-cygwin
runtime! plugin/fakeclip.vim
call vspec#hint({
\   'scope': 'fakeclip#_local_variables()',
\   'sid': 'fakeclip#_sid_prefix()',
\ })
function s:describe__s_read_clipboard__on_cygwin()  "{{{1
  " FIXME
endfunction




function s:describe__s_write_clipboard__on_cygwin()  "{{{1
  " FIXME
endfunction




" __END__  "{{{1
" vim: filetype=vim foldmethod=marker
