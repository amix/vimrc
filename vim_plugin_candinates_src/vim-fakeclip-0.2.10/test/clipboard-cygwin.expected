==== s_read_clipboard on cygwin

==== s_write_clipboard on cygwin



**** Result ****


0 examples, 0 failures
